### PR TITLE
docs(Foundation/Link): updates documentation template

### PR DESCRIPTION
**Pull request by toncabral88**
## Description
https://jirasoftware.catho.com.br/browse/QTM-361

## Setup
to view the components behavior, run `yarn storybook`

## Review guide
- [ ] Code review
- [ ] Documentation updated

### CODE REVIEW

File: stories/Link/Link.story.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+- [Examples](#examples)
+
+## <a id="importing"></a>Importing
+
+```js
+import { Link } from '@catho/quantum';
+```
+or
+```js
+// This option is recommended in case you want to reduce the size of the package
```
Comment (rapahaeru) [inline stories/Link/Link.story.mdx:32]: =]

**Review by rapahaeru — APPROVED** [review]
## Review guide
- [x] Code review
- [x] Documentation updated

File: stories/Link/Link.story.mdx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+import Link from 'catho/quantum/Link';
+```
+
+## <a id="api"></a>API
+
+<ArgsTable of={Link} />
+
+## <a id="examples"></a>Examples
+
```
Comment (MarcosViniciusPC) [inline stories/Link/Link.story.mdx:41]:
```suggestion
### Default
```

File: stories/Link/Link.story.mdx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+<Canvas>
+  <Story story={Default} />
+</Canvas>
+
```
Comment (MarcosViniciusPC) [inline stories/Link/Link.story.mdx:45]:
```suggestion
### Dark
```

**Review by MarcosViniciusPC — DISMISSED** [review]
[x] Code Review
[x] Documentation

I believe adding captions to each example improves the structure of the documentation

**Review by MarcosViniciusPC — APPROVED** [review]
[x] Code review  
[x] Documentation

**Pull request merged**